### DISCUSSION TITLE
Support of GetMotionVectors method

### DISCUSSION THREAD

**Discussion by cheptil** [RomanArzumanyan/VALI#14] [General]
Hello to all

Sorry if I double issue 13, may be here my question will see sooner

So, does VALI support GetMotionVectors method?

**Comment by RomanArzumanyan** [comment]
Hi @cheptil 

It’s certainly exists in codebase, I just need to revive it and add couple unit tests.

I’ll come back to you a bit later when I have a feature branch with fix.

Thanks!
